firmware/controllers/algo/engine_configuration.cpp

```diff
@@ -431,20 +431,41 @@ static void setDefaultWarmupIdleCorrection(DECLARE_CONFIG_PARAMETER_SIGNATURE) {
 }
 
 static void setDefaultWarmupFuelEnrichment(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
-	initTemperatureCurve(WARMUP_CLT_EXTRA_FUEL_CURVE, 1.0);
```

Comment:
Are you sure you want to remove this line? You set only 12 values with copyArrayPartial() but CLT_CURVE_SIZE is 16...